static/src/stylesheets/amp/_hosted.scss

```diff
@@ -442,3 +442,14 @@ $badge-width: $gs-column-width + $gs-gutter;
 .content__hosted-body {
     @include content-gutter();
 }
+
+.content__meta-container {
```

Comment:
I am really glad you chose not to import the entire file! The code needed was really small. Well done 👍